src/ngraph/pass/assign_placement.cpp

```diff
@@ -26,8 +28,31 @@ pass::AssignPlacement::AssignPlacement(function<Placement(shared_ptr<Node>)> pla
 {
 }
 
+pass::AssignPlacement::AssignPlacement(vector<shared_ptr<runtime::Backend>> placement_backends)
+    : m_placement_backends(placement_backends)
+{
+}
+
 bool pass::AssignPlacement::run_on_node(shared_ptr<Node> node)
 {
-    node->set_placement(m_placement_policy(node));
+    if (!m_placement_backends.empty())
+    {
+        size_t backend_indx = 0;
+        for (auto backend : m_placement_backends)
+        {
+            backend_indx += 1;
+            if (backend->is_supported(*node))
+            {
+                node->set_placement(backend_indx);
+
+                return false;
+            }
+        }
+    }
+    else
+    {
+        node->set_placement(m_placement_policy(node));
+    }
+
     return false;
```

Comment:
so it's always returning `false` for now? Regardless of the fact if we find a supported backend?